lib/jobs/uefi-job.js

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2016, EMC, Inc.
```

Comment:
I have tried to clone @keedya 's code to my local, the jshint doesn't report such error. 
I also did some search, seems this is probably due to the incorrect file encoding, change the file-encoding to utf-8 will resolve this issue.
@keedya , please check whether it is due to the file-encoding. If it is a real false-alarm, I think we can open an issue for houndci.